DcShare/Helper/ShareAttachment.swift

```diff
@@ -120,7 +120,7 @@ class ShareAttachment {
                 self.messages.append(msg)
                 self.delegate?.onAttachmentChanged()
                 if self.imageThumbnail == nil {
-                    self.imageThumbnail = result
+                    self.imageThumbnail = result.scaleDownImage(toMax: 96)
```

Comment:
what exactly does this `96` mean? max width/height in pixel? would be nice to have some comment at the corresponding function (not really an issue of this pr).

Reply:
I replaced the hard coded value by `self.thumbnailSize` which also helps to understand where the value comes from.